src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/adjudications_by_finding_code.json

```diff
@@ -33,13 +33,13 @@
       "agencyId": "LEI",
       "description": "Leeds",
       "agencyType": "INST",
-      "active": false
+      "active": true
```

Comment:
what has happened here?

Reply:
Yes looks like it didn't map property.  Let me check

Reply:
Yes not mapped and I've change the default builder to true, will correct code.